js/app-functions/greetings.js

```diff
@@ -0,0 +1,21 @@
+export const showGreeting = () => {
+  const greetingsText = document.querySelector("#greetingsText");
+  const greeting = selectGreeting();
+  greetingsText.textContent = greeting;
+};
+
+const selectGreeting = () => {
+  let randomNum = Math.random();
+  let greeting = "";
+
+  if (randomNum < 0.25) {
+    greeting = "Hello There!";
+  } else if (randomNum < 0.5) {
+    greeting = "Bonjour!";
+  } else if (randomNum < 0.75) {
+    greeting = "Hola!";
+  } else {
+    greeting = "Hey, what's up?";
+  }
+  return greeting;
```

Comment:
An alternative approach might be something like 
```js
 const greetings = ["Hey", "Hola", "Hi"};
 const randomIndex = Math.floor(Math.random() * greetings.length); 
 return greetings[randomIndex]
```

Reply:
Thanks Alex, I was able to implement this successfully. The only change I had to make was replacing "const randomIndex" with "let randomIndex", so the greeting wouldn't be the same each time.